product_docs/docs/pgd/5/cli/installing_cli.mdx

```diff
@@ -13,22 +14,59 @@ By default, Trusted Postgres Architect installs and configures PGD CLI on each P
 
 PGD CLI is installable from the EDB repositories, which you can access with your EDB account. PGD users and BigAnimal users, including those on a free trial, have an EDB account and access to PGD CLI.
 
- These repositories require a token to enable downloads from them. To obtain your token, log in to [EDB Repos 2.0](https://www.enterprisedb.com/repos-downloads). If this is your first time visiting the EDB Repos 2.0 page, you must select **Request Access** to generate your token. Once a generated token is available, select the **Copy** icon to copy it to your clipboard, or select the eye icon to view it.
+### Obtain your EDB subscription token
```

Comment:
Unfortunately, this was trigged by a customer being unable to handle adding a repo safely, and currently, the split between manual install and TPA (and soon BigAnimal and PGD-K) means there will be people arriving at this particular page without that background. It;s likely this will be the only incoming page where that applies so we're not worried about maintaining it.